ibmdbpy/tests/test_base_connexion.py

```diff
@@ -34,10 +34,7 @@ def test_idadb_attr(self, idadb):
         assert isinstance(idadb.data_source_name, six.string_types)
         assert isinstance(idadb._con_type, six.string_types)
         assert isinstance(idadb._idadfs, list)
-        if idadb._con_type == 'odbc':
-            assert isinstance(idadb._connection_string, six.string_types)
-        elif idadb._con_type == 'jdbc':
-            assert isinstance(idadb._connection_string, list)
+        assert isinstance(idadb._connection_string, six.string_types)
```

Comment:
Please revert this change, it doesn't work for JDBC connections:

```
self = <test_base_connexion.Test_ConnectToDB object at 0x7f75f81609e8>
idadb = <ibmdbpy.base.IdaDataBase object at 0x7f76af645470>

    def test_idadb_attr(self, idadb):
        assert isinstance(idadb.data_source_name, six.string_types)
        assert isinstance(idadb._con_type, six.string_types)
        assert isinstance(idadb._idadfs, list)
>       assert isinstance(idadb._connection_string, six.string_types)
E       AssertionError: assert False
E        +  where False = isinstance(['jdbc:db2://<hostname>:50000/BLUDB', 'bluadmin', '<password>'], (<class 'str'>,))
E        +    where ['jdbc:db2://<hostname>:50000/BLUDB', 'bluadmin', '<password>'] = <ibmdbpy.base.IdaDataBase object at 0x7f76af645470>._connection_string
E        +    and   (<class 'str'>,) = six.string_types

test_base_connexion.py:37: AssertionError
```

Reply:
The type of the connection string can be both - a list or a string - depending on the jaydebeapi version. I've modified the testcase to accept both types for jdbc connection strings with commit https://github.com/ibmdbanalytics/ibmdbpy/pull/56/commits/5be5189df0f278f1aff012f45b3ca7bfbaa69554